/// Files: drivers/net/can/usb/mcba_usb.c
/// Fix: 4d6636498c41891d0482a914dd570343a838ad79
/// Fixes: 51f3baad7de943780ce0c17bd7975df567dd6e14

virtual detect

@err exists@
identifier priv;
position p;
@@

mcba_usb_disconnect(...)
{
	...
*	free_candev(priv->netdev);
*	mcba_urb_unlink(priv@p);
	...
}

@script:python depends on detect@
p << err.p;
@@

coccilib.report.print_report(p[0], 'ERROR: CVE-2019-19529')
